services/subnet_manager/src/test/java/com/futurewei/alcor/subnet/SubnetControllerTests.java

```diff
@@ -166,7 +168,7 @@ public void createSubnetState_canNotFindRoute_notPass () throws Exception {
                     .andExpect(MockMvcResultMatchers.jsonPath("$.subnet.id").value(UnitTestConfig.subnetId));
         }catch (Exception ex) {
             //System.out.println(ex.getMessage());
-            assertEquals(UnitTestConfig.createException, ex.getMessage());
+            assertEquals(UnitTestConfig.createException2, ex.getMessage());
```

Comment:
Let us be a little creative in term of naming :-)

Reply:
Okay ~ Naming is one of my weaknesses ~